qiskit_nature/drivers/base_driver.py

```diff
@@ -18,10 +18,10 @@
 from abc import ABC, abstractmethod
 
 from enum import Enum, EnumMeta
-import warnings
 
 from .molecule import Molecule
 from ..exceptions import QiskitNatureError
+from ..deprecation import DeprecatedType, warn_deprecated_same_type_name
 
 
 class DeprecatedEnum(Enum):
```

Comment:
Now that we have `qiskit_nature.deprecation`, this class and the meta-classes below would seem to fit there better, too, don't you think?

Reply:
You are right, I will move those.

Reply:
I moved them